app/navigation/Navigator.scala

```diff
@@ -61,8 +61,6 @@ class Navigator @Inject() () {
     case ProofOfAuthorityPage                       => _ => routes.BankDetailsController.onPageLoad(NormalMode)
     case CheckYourAnswersPage                       => _ => routes.ConfirmationController.onPageLoad()
     case AmendCheckYourAnswersPage                  => _ => routes.AmendConfirmationController.onPageLoad()
-    case ReferenceNumberPage                        => _ => routes.AmendCaseResponseTypeController.onPageLoad(NormalMode)
-    case AmendCaseResponseTypePage                  => getAmendCaseResponseType
     case AmendCaseSendInformationPage               => _ => routes.AmendCaseSendInformationController.showFileUploaded(NormalMode)
```

Comment:
CLARIFICATION: Same as above, is AmendCaseSendInformationPage = AmendCaseSendInformationController still supposed to be here (rather than in the new AmendCaseNavigator ?

Reply:
Same

Reply:
Removed unused code from old navigator